services/booking-api/lambdas/getHistoricalAttendees.js

```diff
@@ -0,0 +1,39 @@
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  console.log('EVENT: ', JSON.stringify(event, undefined));
+
+  let referenceCode = event.pathParameters?.referenceCode;
+
+  if (!referenceCode) {
+    return response.failure({
+      status: 403,
+      message: 'Missing required parameter: "referenceCode"',
+    });
+  }
+
+  referenceCode = decodeURIComponent(referenceCode);
+  const { startTime = '', endTime = '' } = event.queryStringParameters;
+
+  if (!startTime | !endTime) {
```

Comment:
Have a somewhat hard time to understand this expression. Maybe it could be extraced to an function that's describes the check in a clear way?

Reply:
Lets have a discussion about this one